config/apparmor.d/usr.sbin.cups-browsed

```diff
@@ -0,0 +1,20 @@
+#include <tunables/global>
+
+/usr/sbin/cups-browsed flags=(attach_disconnected) {
+  #include <abstractions/base>
+  #include <abstractions/nameservice>
+  #include <abstractions/cups-client>
+  #include <abstractions/dbus>
+  #include <abstractions/p11-kit>
+
+  /{var/,}etc/cups/cups-browsed.conf r,
+  /{var/,}etc/cups/lpoptions r,
+  /{var/,}etc/cups/ppd/* r,
+  /{var/,}run/cups/certs/* r,
```

Comment:
lines changed from default, adding `{var/,}` to enable CUPS to access the new location.